thirtyinch/src/main/java/net/grandcentrix/thirtyinch/TiPresenter.java

```diff
@@ -288,25 +323,18 @@ public String toString() {
     }
 
     /**
-     * when calling {@link #wakeUp()} the presenter can start communicating with the {@link
-     * TiView}.
+     * The view is now attached and ready to receive events. {@link #getView()} is not guaranteed
```

Comment:
Better Javadoc:
`getView() may be null`.
Current javadoc is very confused and have to be read more than once ;)